src/main/scala/com/codacy/client/stash/service/ProjectServices.scala

```diff
@@ -10,17 +10,23 @@ class ProjectServices(client: StashClient) {
   /**
     * Only projects for which the authenticated user has the PROJECT_VIEW permission will be returned.
     */
-  def findById(projectKey: String): RequestResponse[Project] = {
-    client.execute(Request(s"$BASE/$projectKey", classOf[Project]))()
+  def findById(projectKey: String, includeAvatar: Boolean = false): RequestResponse[Project] = {
+    val params = if (includeAvatar) Map("avatarSize" -> "64") else Map.empty[String, String]
+
+    client.execute(Request(s"$BASE/$projectKey", classOf[Project]))(params)
   }
 
   /**
     * Only projects for which the authenticated user has the PROJECT_VIEW permission will be returned.
     */
-  def findAll(pageRequest: Option[PageRequest]): RequestResponse[Seq[Project]] = pageRequest match {
-    case Some(pageRequest) =>
-      client.executePaginatedWithPageRequest(Request(BASE, classOf[Seq[Project]]), pageRequest = pageRequest)()
-    case None => client.executePaginated(Request(BASE, classOf[Seq[Project]]))()
+  def findAll(pageRequest: Option[PageRequest], includeAvatar: Boolean = false): RequestResponse[Seq[Project]] = {
+    val params = if (includeAvatar) Map("avatarSize" -> "64") else Map.empty[String, String]
```

Comment:
since this is duplicated maybe we should have a private function to get the default parameter for the avatarSize or something.

Also, we are repeating the default size for the avatar as 64 in multiple places, should we add something in a common place like here https://github.com/codacy/stash-scala-client/tree/master/src/it/scala/com/codacy/client/stash/helpers ?